xinference/model/embedding/core.py

```diff
@@ -118,7 +118,12 @@ def get_cache_status(
 
 
 class EmbeddingModel:
-    def __init__(self, model_uid: str, model_path: str, device: Optional[str] = None):
+    def __init__(
+        self,
+        model_uid: str,
+        model_path: Optional[str] = None,
```

Comment:
`model_path` here cannot be `Optional[str]`. It must be `str`.
In the upper logic, either the custom path or the path obtained through cache must be a non-None value by this point.